build.gradle.kts

```diff
@@ -15,18 +15,19 @@ val commitHash = kotlin.run {
 	output.trim()
 }
 
-val pluginComingVersion = "0.10.1"
+val pluginComingVersion = "0.10.2"
 val pluginVersion = if (isCI) "$pluginComingVersion-$commitHash" else pluginComingVersion
 val packageName = "org.ice1000.tt"
 
 group = packageName
 version = pluginVersion
 
 plugins {
+	id("com.github.ben-manes.versions") version "0.38.0"
```

Comment:
Why this?

Reply:
This plugin provides `dependencyUpdates` task that lists all the dependencies (including Gradle plugins and Gradle itself) that have later versions than the ones used. It simplifies the version upgrades: one doesn't need to check each dependency manually.